### DISCUSSION TITLE
No benefits added after 3 days of approval

### DISCUSSION THREAD

**Discussion by Oreek** [community/community#141701] [GitHub Education]
### Hi everyone,

I had submitted my application for GitHub students Developer Pack on 14th October and it had been approved in about 1-2 hours (green colored). But I have not yet received the benefits even though it has been more than the maximum 3 days they have mentioned in their FAQs.

**Comment by davevad93** [comment]
Hi @Oreek ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Oreek was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
